src/services/iasql.ts

```diff
@@ -4,6 +4,7 @@ const exec = promisify(execNode);
 
 import { createConnection, } from 'typeorm'
 import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions'
+import { snakeCase, } from 'typeorm/util/StringUtils'
```

Comment:
Btw, did you know that `typeorm` has both a `util` directory and a `utils` directory? WTF!?